### ISSUE TITLE
--skip-updates flag to new

### ISSUE THREAD

**Issue by glycerine** [gobuffalo/buffalo#482]
reference https://groups.google.com/forum/#!topic/golang-nuts/bCBCsSEQVCI

Suggestion: `buffalo new` may benefit from a `--skip-updates` flag -- if this error in fetching golang.org/x/tools turns out to be common. For the life of me, I can't figure out how to make `buffalo new` work on a friend's mac laptop. There's some kind of weird git misconfig going on that is causing `buffalo new` to fail during go get:

~~~
  $ buffalo new -v deal
Buffalo version v0.9.0

--> go get -u golang.org/x/tools/cmd/goimports
# cd /Users/jaten/go/src/golang.org/x/tools; git pull --ff-only
fatal: remote error: 


Invalid authentication credentials.

Please generate a new identifier:
  https://go.googlesource.com/new-password


package golang.org/x/tools/cmd/goimports: exit status 1
--> goimports -w .
Usage:
  buffalo new [name] [flags]

Flags:
      --api                  skip all front-end code and configure for an API server
      --ci-provider string   specify the type of ci file you would like buffalo to generate [none, travis, gitlab-ci] (default "none")
      --db-type string       specify the type of database you want to use [postgres, mysql, sqlite3] (default "postgres")
      --docker string        specify the type of Docker file to generate [none, multi, standard] (default "multi")
  -f, --force                delete and remake if the app already exists
  -h, --help                 help for new
      --skip-dep             skips adding github.com/golang/dep to your app
      --skip-pop             skips adding pop/soda to your app
      --skip-webpack         skips adding Webpack to your app
  -v, --verbose              verbosely print out the go get/install commands
      --with-yarn            allows the use of yarn instead of npm as dependency manager

Error: exit status 1

$ go get -u golang.org/x/tools/cmd/goimports
# cd /Users/jaten/go/src/golang.org/x/tools; git pull --ff-only
fatal: remote error: 


Invalid authentication credentials.

Please generate a new identifier:
  https://go.googlesource.com/new-password


package golang.org/x/tools/cmd/goimports: exit status 1
  $ 
~~~

There's a nice --skip-dep flag already, and as I already have golang.org/x/tools manually installed, I could get unstuck if I had a --skip-updates flag too, to avoid applying the `-u` during go get.

Update: there was a ~/.gitcookies file, referenced from ~/.gitconfig, that was messing things up. Still it would be nice not to force a `go get -u` on pinned packages.

**Comment by glycerine** [comment]
Add on thought: in order to be vendored-package friendly, it may be preferred that --skip-updates is actually the default. Doing a `go get -u` is rarely friendly to library that is pegged to a particular version for vendoring and compatibility reasons.

**Comment by stanislas-m** [comment]
Fixed with #1236.
